core/src/main/scala/org/scalafmt/util/TokenOps.scala

```diff
@@ -65,12 +65,11 @@ object TokenOps {
   }
 
   def isTopLevelStatment(tok: Token, owner: Tree): Boolean = {
-      tok.is[KwObject] ||
-        tok.is[KwDef] ||
-        tok.is[KwCase] ||
-        tok.is[KwPackage] ||
-        owner.parent.exists(_.is[Defn.Class]) ||
-        owner.parent.exists(_.is[Defn.Trait])
+    tok.is[KwObject] || owner.parent.exists(_.is[Defn.Object]) ||
```

Comment:
no the test fails without it:

```
package b {
       object a {
         val v1 = 1
         val v2 = 2
``` 

there should be whitespace between { and object in this case